politeiad/backend/gitbe/gitbe.go

```diff
@@ -457,9 +464,14 @@ func mdFilename(path, id string, mdID int) string {
 // backend.MetadataStream that is completely filled out.
 //
 // This function must be called with the lock held.
-func loadMDStreams(path, id string) ([]backend.MetadataStream, error) {
-	// Get dir.
-	dir := joinLatest(path, id)
+func loadMDStreams(path, id, version string) ([]backend.MetadataStream, error) {
+	var dir string
+	if version == "" {
```

Comment:
you seem to be repeatin this logic above, consider extracting to a function to make a more modular/testable code

Reply:
in fact, can it be a default parameter maybe?